subprojects/signing/src/main/java/org/gradle/plugins/signing/SigningExtension.java

```diff
@@ -232,6 +234,25 @@ public void useGpgCmd() {
         setSignatories(new GnupgSignatoryProvider());
     }
 
+    /**
+     * Use the supplied ascii-armored in-memory PGP secret key and password
+     * instead of reading it from a keyring.
+     *
+     * <pre><code>
+     * signing {
+     *     def secretKey = System.getenv("MY_SECRET_KEY")
```

Comment:
⭕️ We should use project properties here instead of `System.getenv`. Users can use environment variables prefixed with `ORG_GRADLE_PROJECT` to set a project property.

Reply:
Done in 11c4a20.